Firefox currently does not start for some reason if
firefox-extension-websites-integration is installed.
diff -Nru lightdm-remote-session-uccsconfigure-1.1.orig/firefox-uccsconfigure.sh lightdm-remote-session-uccsconfigure-1.1/firefox-uccsconfigure.sh
--- lightdm-remote-session-uccsconfigure-1.1.orig/firefox-uccsconfigure.sh	2013-01-08 09:37:58.860678225 -0500
+++ lightdm-remote-session-uccsconfigure-1.1/firefox-uccsconfigure.sh	2013-01-08 09:38:21.520558664 -0500
@@ -1,4 +1,5 @@
 #!/bin/sh
 
-firefox -no-remote https://uccs.landscape.canonical.com/
+#firefox -no-remote https://uccs.landscape.canonical.com/
+chromium --incognito https://uccs.landscape.canonical.com/
 gnome-session-quit --logout --no-prompt
